.github/workflows/windows_release.yml

```diff
@@ -45,7 +47,7 @@ jobs:
     - name: Configure CMake
       shell: bash
       working-directory: ${{runner.workspace}}/EnergyPlus/build
-      run: cmake -G "Visual Studio 16 2019" -A x64 -DLINK_WITH_PYTHON=ON -DTEX_INTERACTION="batchmode" -DBUILD_FORTRAN=ON -DDOCUMENTATION_BUILD="BuildWithAll" -DMIKTEX_BINARY_PATH="C:/Program Files/MikTeX/miktex/bin/x64" -DBUILD_PACKAGE:BOOL=ON ..
+      run: cmake -G "Visual Studio 16 2019" -A x64 -DLINK_WITH_PYTHON=ON -DTEX_INTERACTION="batchmode" -DBUILD_FORTRAN=ON -DDOCUMENTATION_BUILD="BuildWithAll" -DMIKTEX_BINARY_PATH="C:/Program Files/MikTeX/miktex/bin/x64" -DQTIFWDIR="${{runner.workspace}}/Qt/Tools/QtInstallerFramework/3.2" -DBUILD_PACKAGE:BOOL=ON ..
```

Comment:
I'm not sure if specifying the `QTIFWDIR` is actually doing anything here or not, but it was a step in getting it working, so I'm leaving it for right now.  #bandaid